policy/modules/contrib/boothd.if

```diff
@@ -0,0 +1,39 @@
+## <summary>policy for boothd</summary>
+
+########################################
+## <summary>
+##    Execute boothd_exec_t in the boothd domain.
```

Comment:
In this file, spaces are used for formatting. I think it (currently) is recommended to use tabs.